docs/dev-guide/api-ref.md

```diff
@@ -784,3 +784,99 @@ returns 200 ok with `{ master: <core-uptime>, mediators: { <urn>: <mediator-upti
 Returns the server uptime in seconds. Includes a list of all registered mediators and if heartbeats have been received for them, will include their uptimes as well.
 
 Note that this is a public endpoint that does not require any authorization. It is convenient for integrating with external monitoring tools.
+
+### Metadata resource
+
+The metadata resource allows the user to export and import Channels, Clients, Mediators, Users and ContactGroups.
+
+#### Export Metadata
```

Comment:
Perhaps add a little more about when each how each API endpoint should be used. I.e. validate before sending the main import request.